Sources/GRPCReflectionService/Service/ReflectionService.swift

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright 2024, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+private import DequeModule
+public import GRPCCore
+public import SwiftProtobuf
+
+#if canImport(FoundationEssentials)
+public import struct FoundationEssentials.URL
+public import struct FoundationEssentials.Data
```

Comment:
You are half right, it _is_ only used privately but the module can only be imported at one access level so if you `private` the compiler emits a warning saying its redundant because of the `public` import.

Reply:
Oh I see, I thought it was different when importing specific types. Cool, no need to change then.